.travis.yml

```diff
@@ -26,9 +26,6 @@ install:
 script:
   - npm run-script coverage
 
-after_script:
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
I'll see if I can get it working. In the meantime, it might be a good idea to include the COVERALLS_REPO_TOKEN into travis, just to be safe.